src/AbstractSQLRules2SQL.ts

```diff
@@ -939,6 +940,12 @@ const typeRules: Dictionary<MatchFn> = {
 		const date = DateValue(getAbstractSqlQuery(args, 0), indent);
 		return `DATE(${date})`;
 	},
+	DateTrunc: (args, indent) => {
+		checkArgs('DateTrunc', args, 2);
+		const field = Field(getAbstractSqlQuery(args, 0), indent);
+		const str = args[1];
```

Comment:
This 100% needs some form of validation rather than just dumping it into the sql as-is as that opens up all sorts of potential issues, especially if it comes directly from a user, using
```suggestion
		const str = TextValue(args[1], indent);
```
should be able to handle that